examples/extensions/add_site_links.py

```diff
@@ -0,0 +1,219 @@
+    extension_feed_item2 = extension_feed_item_operation2.create
+    extension_feed_item2.extension_type = extension_type_enum.SITELINK
+    extension_feed_item2.sitelink_feed_item.link_text.value = (
+        'Thanksgiving Specials')
+    extension_feed_item2.start_date_time.value = date_range.start_datetime
+    extension_feed_item2.end_date_time.value = date_range.end_datetime
+    # Targets this sitelink for the United States only.
+    # A list of country codes can be referenced here:
+    # https://developers.google.com/adwords/api/docs/appendix/geotargeting
+    united_states = geo_target_constant_service.geo_target_constant_path(2048)
+    extension_feed_item2.targeted_geo_target_constant.value = united_states
+    final_url2 = extension_feed_item2.sitelink_feed_item.final_urls.add()
+    final_url2.value = 'http://www.example.com/thanksgiving'
+
+    extension_feed_item_operation3 = client.get_type(
+        'ExtensionFeedItemOperation', version='v2')
+    extension_feed_item3 = extension_feed_item_operation3.create
+    extension_feed_item3.extension_type = extension_type_enum.SITELINK
+    extension_feed_item3.sitelink_feed_item.link_text.value = 'Wifi available'
+    extension_feed_item3.targeted_campaign.value = campaign_resource_name
+    extension_feed_item3.device = feed_item_target_device_enum.MOBILE
+    final_url3 = extension_feed_item3.sitelink_feed_item.final_urls.add()
+    final_url3.value = 'http://www.example.com/mobile/wifi'
+
+    extension_feed_item_operation4 = client.get_type(
+        'ExtensionFeedItemOperation', version='v2')
+    extension_feed_item4 = extension_feed_item_operation4.create
+    extension_feed_item4.extension_type = extension_type_enum.SITELINK
+    extension_feed_item4.sitelink_feed_item.link_text.value = 'Happy hours'
+    extension_feed_item4.targeted_campaign.value = campaign_resource_name
+    extension_feed_item4.device = feed_item_target_device_enum.MOBILE
+    final_url4 = extension_feed_item4.sitelink_feed_item.final_urls.add()
+    final_url4.value = 'http://www.example.com/happyhours'
+    _populate_ad_schedule(
+        extension_feed_item4.ad_schedules.add(), day_of_week_enum.MONDAY, 18,
+        minute_of_hour_enum.ZERO, 21, minute_of_hour_enum.ZERO)
+    _populate_ad_schedule(
+        extension_feed_item4.ad_schedules.add(), day_of_week_enum.TUESDAY, 18,
+        minute_of_hour_enum.ZERO, 21, minute_of_hour_enum.ZERO)
+    _populate_ad_schedule(
+        extension_feed_item4.ad_schedules.add(), day_of_week_enum.WEDNESDAY, 18,
+        minute_of_hour_enum.ZERO, 21, minute_of_hour_enum.ZERO)
+    _populate_ad_schedule(
+        extension_feed_item4.ad_schedules.add(), day_of_week_enum.THURSDAY, 18,
+        minute_of_hour_enum.ZERO, 21, minute_of_hour_enum.ZERO)
+    _populate_ad_schedule(
+        extension_feed_item4.ad_schedules.add(), day_of_week_enum.FRIDAY, 18,
+        minute_of_hour_enum.ZERO, 21, minute_of_hour_enum.ZERO)
```

Comment:
maybe make this into a loop iterating on the day of week enum (since the rest is the same), so less repeated code?

Reply:
Good idea.